build/glutenblocks.style.css

```diff
@@ -1 +1,311 @@
\ No newline at end of file
+.gb-button-group {
+  display: inline-block;
```

Comment:
Did you ran the compilation using `yarn production` ?

Reply:
Nope, I will add the new compiled assets.